orc8r/cloud/go/services/state/indexer/reindex/reindexer_singleton_test.go

```diff
@@ -115,6 +113,9 @@ func TestSingletonRun(t *testing.T) {
 	recvCh(t, ch)
 	recvCh(t, ch)
 	recvCh(t, ch)
+
+	// Unregister indexers
+	register(t)
```

Comment:
Instead of doing this here, let's register at the beginning of your changes (below)